### DISCUSSION TITLE
Access to Slack channel

### DISCUSSION THREAD

**Discussion by nikswamy** [FStarLang/FStar#2518] [General]
Many F* users and developers chat on a Slack channel called #fstar-hackery.

You can usually join this channel automatically, by going to [JoinEverestSlack](https://aka.ms/JoinEverestSlack). 

However, Slack's public joining link expires periodically and needs to be refreshed: so, you may find that link doesn't work.

If that happens, please comment on this thread tagging @nikswamy @aseemr @tahina-pro

**Comment by nikswamy** [comment]
While we intend GitHub Discussions to be the primary way to engage with the F* community, we also use a Slack channel for the following purposes:

 * It is integrated with our CI system and we receive build notifications etc. here. So, if you're opening PRs on F*, you might want to join the slack to see build notifications.

 * Short discussions about things that are maybe not clearly of general interest could start on Slack
 
 * Slack also provides a way to have conversations with other F* users and developers in smaller groups and private channels

**Comment by noonien** [comment]
It seems that the slack requires a @microsoft.com or @google.com domain. Can only people from those organizations join the slack?

**Reply by nikswamy** [reply]
Can you please try the link on the fstar-lang.org front page again? It is https://aka.ms/JoinEverestSlack
The joining link expired and I had to update it

**Reply by noonien** [reply]
This worked, thanks!

**Comment by btj** [comment]
The link has expired again

**Reply by nikswamy** [reply]
updated again

**Comment by dears1rr** [comment]
The slack requires again only the @microsoft.com or @google.com domain for the email to join. I guess that the link is down again, could you please update it? Thank you.

**Reply by nikswamy** [reply]
fixed again

**Reply by dears1rr** [reply]
Hello, it seems that I'm still required to use a google email account to sign in; would it be possible to use another domain name? Thank you

**Reply by nikswamy** [reply]
Terribly sorry about this ... I've updated the link again. Can you try https://aka.ms/JoinEverestSlack again, please?

**Comment by shonfeder** [comment]
Hello! I am sorry to bother you, @nikswamy, but it seems the link is stale again. If the slack is still a place for people interested in hacking with and on F* to connect for some things, I'd be keen to join.

**Reply by nikswamy** [reply]
I've updated it ... please try again.

**Comment by AlexHodder** [comment]
Hi @nikswamy, would you be able to refresh the link again? Having the same issue as previous users and seems the link is stale. Many thanks!

**Reply by nikswamy** [reply]
refreshed

**Comment by dariusf** [comment]
Hi @nikswamy, the link has expired again.

**Reply by nikswamy** [reply]
I just refreshed the link. It seems it is now finally possible to get slack to set the link to never expire. So, hopefully this should not be a problem again. I hope it works!

**Reply by dariusf** [reply]
It did, thank you!
